src/Microsoft.AspNetCore.Mvc.Formatters.Xml/XmlDataContractSerializerOutputFormatter.cs

```diff
@@ -177,6 +177,26 @@ protected virtual DataContractSerializer CreateSerializer(Type type)
             return XmlWriter.Create(writer, xmlWriterSettings);
         }
 
+        /// <summary>
+        /// Creates a new instance of <see cref="XmlWriter"/> using the given <see cref="TextWriter"/> and
+        /// <see cref="XmlWriterSettings"/>.
+        /// </summary>
+        /// <param name="context">The formatter context associated with the call.</param>
+        /// <param name="writer">
+        /// The underlying <see cref="TextWriter"/> which the <see cref="XmlWriter"/> should write to.
+        /// </param>
+        /// <param name="xmlWriterSettings">
+        /// The <see cref="XmlWriterSettings"/>.
+        /// </param>
+        /// <returns>A new instance of <see cref="XmlWriter"/></returns>
```

Comment:
Missing that final period here and in the`XmlSerializerOutputFormatter` version.